src/photos/ducks/albums/components/AlbumsView.jsx

```diff
@@ -20,19 +21,21 @@ const Content = ({ list }) => {
   }
 }
 
-export default class AlbumsView extends Component {
-  render() {
+class AlbumsView extends Component {
+  render({ t }) {
     if (this.props.children) return this.props.children
     if (!this.props.albums) {
       return null
     }
     return (
       <div className={styles['pho-content-wrapper']}>
         <Topbar viewName="albums">
-          <AlbumsToolbar />
+          <AlbumsToolbar t={t} />
```

Comment:
Why pass `t` here and not using `translate` HOC for the `AlbumToolbar` too? Am I missing something?

Reply:
Yeah but it's easy to miss: The `Topbar` may wrap the `AlbumsToolbar` inside the cozy-bar. If that happens, `t` injected via the HOC will have the bar's translation context, so we need to pass it an instance of `t` that is bound to the correct context.